### DISCUSSION TITLE
How can I get the URL of API endpoint that was created ?

### DISCUSSION THREAD

**Discussion by DeerajTheepshi** [aws/aws-cdk#23461] [Q&A]
I want to create a API Gateway for my service. Once the service is deployed through CDK, I want to register the URL of the API to another service by making a POST call. What is the best way to do this ? How to get the URL and where to make the POST call ?

**Comment by peterwoodworth** [comment]
How to get the URL will differ depending on which exact constructs you are using. `RestApi` will create a `CfnOutput` that outputs the URL by default, but you might have to manually configure this if using other constructs.

> where to make that POST call?

I'm not sure what exactly your use case is - but a simple solution that comes to mind is having a shell script which runs the CDK deploy, then makes the POST call using the value returned from the CfnOutput? Or do you want the POST call to happen during deployment but after the service is created?

**Reply by Gamehu** [reply]
i have this issue either,i want to make that POST call from other project like react-frontend?thanks
